Prose/ProseLib/Sources/TestHelpers/XCTestCase+Combine.swift

```diff
@@ -0,0 +1,91 @@
+import Combine
+import XCTest
+
+// Source: https://www.swiftbysundell.com/articles/unit-testing-combine-based-swift-code/
```

Comment:
Could you add this as `/// - Copyright:` on `await`? This way it appears in Quick Help.

Reply:
And it's important that you see the copyright in quick help?